src/path_info.h

```diff
@@ -4,6 +4,8 @@
 
 #include <string>
 
+#include "enums.h"
```

Comment:
```suggestion
enum class holiday : int;
```
Forward declaration is enough in the header. The "enums.h" header is only needed within the cpp file.